content/themes/casper

```diff
@@ -1 +1 @@
-Subproject commit ede6b03afad148eadba701bce9e65401d9e034e8
+Subproject commit c1443a6e04dccdbbe8d138af4cb36410f29759e6
```

Comment:
This commit shouldn't be here - it's effectively downgrading casper from 1.2.5 to 1.2.4.

If you can please undo this, I think the PR is good to go